static/cast/README.md

```diff
@@ -0,0 +1,31 @@
+## Cast file
+
+The file ended with `.cast` comes from [Asciinema](https://asciinema.org/) library which records and displays terminal sessions. 
```

Comment:
The file ended with `.cast` comes from [Asciinema](https://asciinema.org/) library which records and displays terminal sessions.   -->

The `.cast` files are downloaded from  [Asciinema](https://asciinema.org/), they are used by developers to record and replay terminal sessions.